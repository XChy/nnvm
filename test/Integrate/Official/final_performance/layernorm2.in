version https://git-lfs.github.com/spec/v1
oid sha256:55db4e6422757e53631fcebb783b898be884535bac6ed8f156fd9f1cb1d41325
size 5906645
