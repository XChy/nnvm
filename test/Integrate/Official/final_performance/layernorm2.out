0x1.207a4p-6
0